CODIGO/Declares.bas

```diff
@@ -785,35 +805,54 @@ Enum eModoHechizos
     SinBloqueo
 End Enum
 
+Enum e_BuffSate
+    eNormal = 0
+    eMinBuff = 1
+    eHighBuff = 2
+    eBlink = 3
+End Enum
+
+Public Type t_UserStats
+    MaxHp As Integer
+    MinHp As Integer
+    HpShield As Long
+    maxman As Integer
+    minman As Integer
+    MaxSTA As Integer
+    MinSTA As Integer
+    MaxAGU As Byte
+    MinAGU As Byte
+    MaxHAM As Byte
+    MinHAM As Byte
+    GLD As Long
+    OroPorNivel As Long
```

Comment:
Lets use english
`GoldPerLevel`

Reply:
same as above this variablers already exist, sometimes I do the extra work of fixing old spanish names but I dont want to over extend, this is already a big change